### PR TITLE
Add last updated date to search results

### PR DESCRIPTION
**Pull request by joel-wright**
# Description

This PR adds last_updated_date to search results.

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

## How Has This Been Tested?

Extended tests.

## Reviewer Checklist

- [ ] The PR represents a single feature (small driveby fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

File: app/core/ingestion/ingest_row.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def _parse_str(cls, key: str, value: str, field_info: dict[str, type]) -> Any:
             return value
 
         if field_info[key] == datetime:
-            return datetime.strptime(value, "%Y-%m-%d")
+            return datetime.strptime(value, "%Y-%m-%d").replace(tzinfo=timezone.utc)
```
Comment (diversemix) [inline app/core/ingestion/ingest_row.py:96]: Is the replace cos we don't care about the time part?

File: tests/core/ingestion/helpers.py [hunk 2]
```diff
@@ -78,8 +78,9 @@
 SLUG_DOCUMENT_NAME = "DocSlug1"
 COLLECTION_IMPORT_ID = "CPR.Collection.1"
 
-FOUR_EVENT_ROWS = """Id,Eventable type,Eventable Id,Eventable name,Event type,Title,Description,Date,Url,CPR Event ID,CPR Family ID,Event Status
+FIVE_EVENT_ROWS = """Id,Eventable type,Eventable Id,Eventable name,Event type,Title,Description,Date,Url,CPR Event ID,CPR Family ID,Event Status
 1101,Legislation,1001,Title1,Passed/Approved,Published,,2019-12-25,,CCLW.legislation_event.1101.0,CCLW.family.1001.0,OK
+1101,Legislation,1001,Title1,Amended,Amended,,2021-12-25,,CCLW.legislation_event.1101.1,CCLW.family.1001.0,OK
```
Comment (diversemix) [inline tests/core/ingestion/helpers.py:83]: Do we also want to test that an additional and later change to title is correctly used?

**Review by diversemix — APPROVED** [review]
LGTM - just a suggestion for another test

File: tests/core/ingestion/helpers.py (hunk 2 above)
Comment (joel-wright) [reply tests/core/ingestion/helpers.py:83]: have modified one of the tests to show last updated date changing as events are added.

**Pull request merged**